bundle-workflow/tests/tests_ci_workflow/test_check.py

```diff
@@ -5,62 +5,7 @@
 # compatible open source license.
 
 import unittest
-from unittest.mock import MagicMock
-
-from ci_workflow.check import Check
-from manifests.input_manifest import InputManifest
-
-
-class DummyTestCheck(Check):
-    def check(self):
-        pass
 
 
 class TestCheck(unittest.TestCase):
-    def setUp(self):
-        self.check = DummyTestCheck(
-            component=MagicMock(),
-            git_repo=MagicMock(),
-            version="1.1.0",
-            arch="x86",
-            snapshot=False,
-        )
-
-    def test_properties(self):
-        self.assertEqual(self.check.arch, "x86")
-        self.assertEqual(self.check.version, "1.1.0")
-        self.assertFalse(self.check.snapshot)
-        self.assertEqual(self.check.opensearch_version, "1.1.0")
-        self.assertEqual(self.check.component_version, "1.1.0.0")
-
-
-class TestCheckSnapshot(unittest.TestCase):
-    def setUp(self):
-        self.check = DummyTestCheck(
-            component=MagicMock(),
-            git_repo=MagicMock(),
-            version="1.1.0",
-            arch="x86",
-            snapshot=True,
-        )
-
-    def test_snapshot_version(self):
-        self.assertTrue(self.check.snapshot)
-        self.assertEqual(self.check.opensearch_version, "1.1.0-SNAPSHOT")
-        self.assertEqual(self.check.component_version, "1.1.0.0-SNAPSHOT")
-
-
-class TestCheckOpenSearch(unittest.TestCase):
-    def setUp(self):
-        self.check = DummyTestCheck(
-            component=InputManifest.Component(
-                {"name": "OpenSearch", "repository": "", "ref": ""}
-            ),
-            git_repo=MagicMock(),
-            version="1.1.0",
-            arch="x86",
-            snapshot=True,
-        )
-
-    def test_component_version(self):
-        self.assertEqual(self.check.component_version, "1.1.0-SNAPSHOT")
+    pass
```

Comment:
Is this removed since there it directly takes in the target now?

Reply:
Right. This class no longer has any properties, just serves as a common constructor that stores `target`. I didn't want to remove the test class though in case we add something useful to it in the future that's testable.